maquetado/index.html

```diff
@@ -9,6 +9,20 @@
   </head>
   <body>
     <main class="main-project">
+      <header class="header-project">
+        <nav class="nav">
+          <ul class="nav-container">
+            <li class="nav-item">
+              <a href="" class="nav-link">
+                <img src="../assets/wolox_logo.svg" alt="wolox_logo" class="nav-img">
+              </a>
+            </li>
+            <li class="nav-item">
+              <span>Developer Tools</span>
+            </li>
+          </ul>
+        </nav>
```

Comment:
This navbar is too overcomplicated for a navbar haha, try to make things as simple as possible, there is no need to be a link on it, and the `ul` is not needed either, toy could just use the nav tag with and img and text.